### PR TITLE
feat(config-ui): support teambition connection config

### LINKED ISSUE #4774: [Feature][Config Ui] Support for adding connection with teambition
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar feature requirement.

### Use case

_No response_

### Description

It needs ui to add teambition connection, just like other plugins.

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by coldgust**
### Summary
Let config ui support teambition connection. Since there is no teambition logo, I use tapd logo instead temporarily.

### Does this close any open issues?
Closes #4774

### Screenshots
Snipaste_2023-03-26_00-28-44

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: config-ui/src/plugins/components/connection-form/operate/test.tsx [hunk 1]
```diff
@@ -39,7 +39,8 @@ export const Test = ({ plugin, values, errors }: Props) => {
 
   const handleSubmit = () => {
     onSubmit(
-      pick(values, ['endpoint', 'token', 'username', 'password', 'app_id', 'secret_key', 'proxy', 'authMethod']),
+      pick(values, ['endpoint', 'token', 'username', 'password', 'app_id', 'secret_key', 'proxy', 'authMethod',
```
Comment (mintsweet) [inline config-ui/src/plugins/components/connection-form/operate/test.tsx:42]: Can you remove the two fields `app_id` and `secret_key`

File: config-ui/src/plugins/register/teambition/assets/icon.svg
Comment (mintsweet) [inline config-ui/src/plugins/register/teambition/assets/icon.svg:1]: @yumengwang03 Can you provide an icon?

File: config-ui/src/plugins/register/teambition/connection-fields/tenant-id.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    setError(value ? '' : 'TenantId is required');
+  }, [value]);
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setValue(e.target.value);
+  };
+
+  return (
+    <FormGroup
+      label={<S.Label>{label ?? 'TenantId'}</S.Label>}
```
Comment (mintsweet) [inline config-ui/src/plugins/register/teambition/connection-fields/tenant-id.tsx:76]: `tenant-id.tsx`, only applies to this plugin, so there is no need to `lable` these fields.
You can use it.

```
<S.Label>TenantId<S.Label>
```

File: config-ui/src/plugins/register/teambition/connection-fields/tenant-type.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+import React, { useEffect } from 'react';
+import { FormGroup, InputGroup } from '@blueprintjs/core';
+
+import * as S from './styled';
+
+interface Props {
+  label?: string;
+  subLabel?: string;
+  disabled?: boolean;
+  placeholder?: string;
```
Comment (mintsweet) [inline config-ui/src/plugins/register/teambition/connection-fields/tenant-type.tsx:45]: These fields are also.

**Review by mintsweet — CHANGES_REQUESTED** [review]
Thank you very much for your contribution.

**Comment by likyh** [comment]
Wow, Full Stack Developer~

**Comment by yumengwang03** [comment]
Teamabition
@coldgust Here's the icon~

**Comment by JoshuaPoddoku** [comment]
Thank you for your contribution. Please join our slack: https://join.slack.com/t/devlake-io/shared_invite/zt-18uayb6ut-cHOjiYcBwERQ8VVPZ9cQQw 
and @coldgust, we want to issue you a certificate to honor your contribution.

**Review by mintsweet — APPROVED** [review]
LGTM.

**Review by likyh — APPROVED** [review]

**Comment by coldgust** [comment]
Hi, @JoshuaPoddoku I have joined slack. Thank you for the certificate.

**Comment by JoshuaPoddoku** [comment]
Hi @coldgust Thanks for joining slack! I wanted to know your "full name" so we can process the certificate. I tried reaching out to you on slack. Please let us know. Thanks!

**Pull request merged**